Update Standard Setup in README.md

johnnaegle opened the pull request:
Update setup instructions to include the client id and client secret in the OmniAuth configuration when Devise is not in use.  Add basic information about how to generate the oauth keys if you are running your own canvas-lms install. Remove trailing colons on level 2 headers

johnnaegle committed:
Update setup instructions to include the client id and client secrete in the
OmniAuth configuration when Devise is not in use.  Add basic information about
how to generate the oauth keys if you are running your own canvas-lms install.
Remove trailing colons on level 2 headers

johnnaegle commented:
FWIW, This is what my omniauth initializer looks like now.  I found I didn't have to do the trickery with request.env, but instead of I could set the client options directly.  I wasn't going to make this change to the README.

``` ruby
Rails.application.config.middleware.use OmniAuth::Builder do
  provider :canvas, OMNIAUTH_CONFIG['canvas']['client_id'], OMNIAUTH_CONFIG['canvas']['client_secret'],
  {
    :client_options => {
      :site => APP_CONFIG['canvas_host']
    }
  }
end
```

jbasdf reviewed (commented):
Changes look good. I wonder about including your version of the initializer as another example that doesn't use the env trick? I don't want to confuse users, but if the instructions are clear I think it would benefit users.

johnnaegle committed:
Added alternative setup instructions

Adds the alternative setup instructions and moves some text / makes some formatting changes.
